package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ora",
-	"version": "5.4.1",
+	"version": "6.0.0",
```

Comment:
This should be done by the maintainers when they're ready to release, not in a PR.